src/Nethermind/nuget.config

```diff
@@ -3,14 +3,19 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="system.commandline" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
```

Comment:
feels weird to be using a nightly build feed that is private to Microsoft tbh.

Reply:
I agree. They last updated this package on the NuGet production feed over two years ago. Now, in their repo they indicate this feedwhere updates are rare, not daily. I was convinced to go this way because _the `dotnet` CLI tool uses these versions._